deploy_script.sh

```
@@ -0,0 +1,98 @@
+    chmod 777 $BACKUP_DIR
+}
+
+# Eval with user input
+run_task() {
+    local task=$1
+    eval "$task"
+}
+
+# Temporary file race condition
+create_backup() {
+    local backup_file=/tmp/backup_$(date +%s).tar.gz
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)